Foundation/ProgressFraction.swift

```diff
@@ -9,18 +9,18 @@
 
 // Implementation note: This file is included in both the framework and the test bundle, in order for us to be able to test it directly. Once @testable support works for Linux we may be able to use it from the framework instead.
 
-internal struct _ProgressFraction : Equatable, CustomDebugStringConvertible {
-    var completed : Int64
-    var total : Int64
-    private(set) var overflowed : Bool
+public struct _ProgressFraction : Equatable, CustomDebugStringConvertible {
```

Comment:
I know why you want to do this, but we can't make this public. What other options do we have?